saleor/graphql/notifications/tests/test_external_notification_query.py

```diff
@@ -0,0 +1,508 @@
+import json
+from unittest.mock import patch
+
+import pytest
+from graphql_relay.node.node import to_global_id
+
+from ....account.models import User
+from ....core.notify_events import UserNotifyEvent
+from ....graphql.tests.utils import assert_no_permission
+from ....product.models import ProductVariant
+from .conftest import query_test_data
+
+
+@pytest.mark.parametrize("variables, status_code", query_test_data)
+def test_query(
+    variables,
+    status_code,
+    external_notification_trigger_query,
+    staff_api_client,
+    permission_manage_users,
+):
+    response = staff_api_client.post_graphql(
+        external_notification_trigger_query,
+        variables,
+        permissions=[permission_manage_users],
+        check_no_permissions=False,
+    )
+    assert response.status_code == status_code
+
+
+@patch("saleor.plugins.webhook.plugin.WebhookPlugin.notify")
+@patch("saleor.plugins.admin_email.plugin.AdminEmailPlugin.notify")
+@patch("saleor.plugins.user_email.plugin.UserEmailPlugin.notify")
+@patch("saleor.plugins.sendgrid.plugin.SendgridEmailPlugin.notify")
+def test_notify_sendgrid_via_external_notification_trigger_for_all_plugins(
```

Comment:
As mentioned earlier, no need to mock and include plugins in Saleor tests. You need to only confirm that we called a proper method on PluginsManager.

Reply:
Can we rename the test to be more generic and related to manager's notify instead of Sendgrid?